prow/daily-release-qualification.sh

```diff
@@ -63,5 +63,18 @@ cp -R ${DAILY_BUILD}/install/* install/
 ARTIFACTS_DIR="${GOPATH}/src/github.com/istio-releases/daily-release/_artifacts"
 
 echo 'Running E2E Tests'
-ARGS="--test_logs_path="${ARTIFACTS_DIR}" --istioctl_url "${ISTIOCTL_STAGE_URL}" "$@""
-make e2e_all E2E_ARGS="${ARGS}"
+ARGS=( --test_logs_path="${ARTIFACTS_DIR}" --istioctl_url "${ISTIOCTL_STAGE_URL}" "$@" )
+
+EXTRA_E2E_ARGS=()
+EXTRA_E2E_ARGS+=( --mixer_tag="${TAG}" )
+EXTRA_E2E_ARGS+=( --pilot_tag="${TAG}" )
+EXTRA_E2E_ARGS+=( --proxy_tag="${TAG}")
+EXTRA_E2E_ARGS+=( --ca_tag="${TAG}")
+EXTRA_E2E_ARGS+=( --mixer_hub="${HUB}")
+EXTRA_E2E_ARGS+=( --pilot_hub="${HUB}")
+EXTRA_E2E_ARGS+=( --proxy_hub="${HUB}")
+EXTRA_E2E_ARGS+=( --ca_hub="${HUB}")
+
+go test -v -timeout 20m ./tests/e2e/tests/simple -args ${ARGS[@]} ${EXTRA_E2E_ARGS[@]}
+go test -v -timeout 20m ./tests/e2e/tests/mixer -args ${ARGS[@]} ${EXTRA_E2E_ARGS}
```

Comment:
how is `${EXTRA_E2E_ARGS[@]}` different from `${EXTRA_E2E_ARGS}` (line 78 79)? Can we choose one of them for all go test commands?

Reply:
Nice catch, they all should be with [@].

Only `${EXTRA_E2E_ARGS[@]}` can give you all elements in the array.